packit.yaml

```diff
@@ -46,3 +49,27 @@ jobs:
       - fedora-development
       - centos-stream-8-x86_64
       - centos-stream-9-x86_64
+
+  - job: propose_downstream
+    trigger: release
+    metadata:
+      dist_git_branches:
+        - fedora-development
+        - fedora-35
+        - fedora-36
```

Comment:
@martinpitt Do you want to be explicit here for some reason? (I am just interested since we added two new [aliases](https://packit.dev/docs/configuration/#aliases) recently and I hope people can pick any subset without hardcoding.)